EventListener/CorsListener.php

```diff
@@ -95,7 +95,10 @@ public function onKernelRequest(RequestEvent $event): void
         }
 
         // perform preflight checks
-        if ('OPTIONS' === $request->getMethod() && $request->headers->has('Access-Control-Request-Method')) {
+        if ('OPTIONS' === $request->getMethod() &&
+            ($request->headers->has('Access-Control-Request-Method') ||
+                $request->headers->has('Access-Control-Request-Private-Network'))
```

Comment:
is it allowed to have the `Access-Control-Request-Private-Network` header without `Access-Control-Request-Method`?